package/yast2-registration.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Apr  6 16:10:58 CEST 2016 - schubi@suse.de
+
+- Force refreshing while adding a service (bnc#967828).
```

Comment:
OK, but why? Reading the bug, I conclude "otherwise registration can fail". Well, it may be obvious, but it is still more clear to state it explicitly.